mkFit/MkFinder.cc

```diff
@@ -295,6 +295,7 @@ void MkFinder::SelectHitIndices(const LayerOfHits &layer_of_hits,
 
       const float z  = Par[iI].ConstAt(itrack, 2, 0);
       const float dz = std::abs(nSigmaZ * std::sqrt(Err[iI].ConstAt(itrack, 2, 2)));
+      const float edgeCorr = std::abs(0.5f*(L.m_layer_info->m_rout-L.m_layer_info->m_rin)/std::tan(Par[iI].ConstAt(itrack, 5, 0)));
```

Comment:
it's worth to check if using `r/z` is going to work better here instead of tanTheta. This would avoid promoting significantly "twisted" tracks from being promoted to the edge category (e.g. an update with a hit far away in z will align the track along z and make this correction very large).

Reply:
are you referring to track r/z or layer r/z? I tried the latter in the first version (commit 8b69bbc) with larger fake rates (http://uaf-8.t2.ucsd.edu/~cerati/plots_Sep08_ttbar_testzfix_edge_gcc/plots_building_initialStep/effandfakePtEtaPhi.pdf)
So I am not very optimistic. Also with the commit below the fake rate increase is further reduced

Reply:
I meant the track r/z